src/main/java/com/jalasoft/jfc/model/utility/Md5Checksum.java

```diff
@@ -28,19 +28,19 @@
     private static final Logger logger = Logger.getLogger(Md5Checksum.class.getName());
 
     /**
-     *  This method return a Md5 checksum from a file.
-     * @param file
-     * @return String
+     *  This method return a boolean after to compare checksums.
+     * @param file receive the path of a file to get md5.
+     * @return a boolean after to compare.
      * @throws IOException when is a invalid file.
      */
-    public static String getMd5(String file) throws IOException {
-        String checksum = null;
+    public static boolean getMd5(String file, String md5Client) throws IOException {
```

Comment:
you can use your own type of exception if you have it

Reply:
ok, it is already done